lib/rubocop/cop/style/numeric_predicate.rb

```diff
@@ -157,6 +159,11 @@ def invert
           end
         end
 
+        def negated?(node)
+          parent = node.parent
+          parent&.send_type? && parent&.method?(:!)
```

Comment:
Can you refactor to the following?

```suggestion
          return false unless (parent = node.parent)

          parent.send_type? && parent.method?(:!)
```

Reply:
Updated.